spec/fixtures/node.rb

```diff
@@ -11,14 +11,14 @@ def node(override = {})
     },
     deploy: {
       dummy_project: {
-        database: {
-          adapter: 'postgresql',
-          username: 'dbuser',
-          password: '03c1bc98cdd5eb2f9c75',
-          host: 'dummy-project.c298jfowejf.us-west-2.rds.amazon.com',
-          database: 'dummydb',
-          reaping_frequency: 10
-        },
+        # database: {
+        #   adapter: 'postgresql',
+        #   username: 'dbuser',
+        #   password: '03c1bc98cdd5eb2f9c75',
+        #   host: 'dummy-project.c298jfowejf.us-west-2.rds.amazon.com',
+        #   database: 'dummydb',
+        #   reaping_frequency: 10
+        # },
```

Comment:
The appropriate behavior for the definition of a database in the node should be to override the settings from RDS. As this is the case I commented out the database section. Ultimately the behavior of the scripts should implement this, and new tests should be written to simulate and verify that parameters can be overridden in this way.